src/Symfony/Component/Cache/Adapter/AbstractTagAwareAdapter.php

```diff
@@ -149,6 +149,16 @@ static function ($deferred, &$expiredIds) use ($getId, $tagPrefix) {
      */
     abstract protected function doInvalidate(array $tagIds): bool;
 
+    /**
+     * Returns the tags bound to the provided ids.
+     */
+    protected function doFetchTags(array $ids): iterable
```

Comment:
If `$ids` means "tag IDs" ... we could rename this argument as `$tagIds`, which would also be consistent with the above method `doInvalidate(array $tagIds)`

Reply:
This is really $ids, to get the tags related to a list of item identifiers.